dataset-mover/src/main/scala/com/socrata/datacoordinator/mover/Main.scala

```diff
@@ -411,36 +427,108 @@ object Main extends App {
           }
         }
 
+        // Current state of the world:
+        //   * PG secondaries know about the dataset under its old AND
+        //     new names
+        //   * The old truth and the new truth have copies of the
+        //     dataset (the new one is uncommitted)
+        //   * Both truths have secondary_manifest records
+        //   * Archival secondary and soda-fountain know only the old
+        //     name
+        //   * The dataset is locked against writes
+
+        // First, _before_ committing to the new truth, update the
+        // archival secondary.  This is because when we commit the
+        // truth change, the archival secondary will see its
+        // secondary_manifest record and try to create the dataset, so
+        // we need to tell it exists before that happens.
+        for(asc <- archivalSecondaryClient if stores.contains("archival")) {
+          log.info("Informing archival secondary of the internal name change...")
+          if(dryRun) {
+            log.info("(not really)")
```

Comment:
😆

Reply:
I love this, but from past experience trying to piece together sumo searches, it might be more useful appended to the previous line rather than on its own line

Reply:
This won't be run as a service, so the logs will be next to each other.